docs/connector-development/connector-builder-ui/pagination.md

```diff
@@ -0,0 +1,172 @@
+# Pagination
+
+Pagination is a mechanism used by APIs in which data is split up into "pages" when returning results, so that the entire response data doesn't need to be returned all at once.
+
+The Connector Builder offers a Pagination section which implements the most common pagination methods used by APIs. When enabled, the connector will use the pagination configuration you have provided to request consecutive pages of data from the API until there are no more pages to fetch.
+
+If your API doesn't support pagination, simply leave the Pagination section disabled.
+
+## Pagination methods
+
+Check the documentation of the API you want to integrate to find which type of pagination is uses. Many API docs have a "Pagination" or "Paging" section that describes this.
+
+The following pagination mechanisms are supported in the connector builder:
+
+- [Offset Increment](#offset-increment)
+- [Page Increment](#page-increment)
+- [Cursor Pagination](#cursor-pagination)
+
+Select the matching pagination method for your API and check the sections below for more information about individual methods.
```

Comment:
nit: should we tell them to use low-code or the Python CDK if need another type of pagination? We can also link the reader to their respective docs

Reply:
done 👍